### DISCUSSION TITLE
Plugin Select window

### DISCUSSION THREAD

**Discussion by Pete3030** [apohl79/audiogridder#695] [Ideas]
Hi,

it would be great if there would be an option to reference plugins within the plugin select window by their developer/company name. I find those submenus a little confusing, sometimes I need click in multiple submenus until i find the right plugin. OK, maybe thats just me..;)

Thanks and best regards,
Peter

**Comment by apohl79** [comment]
You can just use the search field.

**Comment by Pete3030** [comment]
The problem is, the more plugins you have, the harder is to remember all the specific names of your plugins, so I usually go like ok it's from that company, click (at least in Logic this is possible) the company and select the plugin (which is kind of a hazzle with acustica plugins, i have to admit, because the folder is so crowded already), but for the rest of vendors this works fine.. So in my mind it goes like: ok I want to use the tape plugin for IK, so I click on the IK folder..

**Reply by apohl79** [reply]
What you can do with the search field is for example

- type "ik" - now you get all plugins with "ik" in their name, their company name etc, like everything from "ik"
- now add a space and type "tape" and you would filter down to every plugin with "ik" and "tape" in the name, company etc, so you might see all "ik" tape machines etc but the list is much shorter

**Comment by Pete3030** [comment]
OK great, I will try that. Thanks!
